build/build.go

```diff
@@ -226,6 +228,9 @@ func toSolveOpt(ctx context.Context, node builder.Node, multiDriver bool, opt Op
 		AllowedEntitlements: opt.Allow,
 		SourcePolicy:        opt.SourcePolicy,
 	}
+	if opt.RawOpts != nil {
+		so.FrontendAttrs = opt.RawOpts
+	}
```

Comment:
Are frontend attributes set here intended to be overridden after?

Reply:
Updated to prioritize `RawOpts` implicit opts